webapp/templates/jury/submission_edit_source_modal.html.twig

```diff
@@ -0,0 +1,24 @@
+<div class="modal fade" tabindex="-1" role="dialog" aria-hidden="true">
+    <div class="modal-dialog modal-lg" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title">
+                    Submission details
+                </h5>
+                <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+            </div>
+            {{ form_start(form) }}
+            <div class="modal-body">
+                {{ form_widget(form) }}
+            </div>
+            <div class="modal-footer">
+                <button type="button" class="btn btn-secondary" data-bs-dismiss="modal">Cancel</button>
+                {# {{ form_row(form.submit, { 'label': 'Submit me' }) }} #}
```

Comment:
```suggestion
```

Reply:
@Kevinjil if you re-add the submit button in the form type and then do
```twig
{{ form_widget(form.submit) }}
```

I think that works. the label can be set in the type as well

Reply:
> @Kevinjil if you re-add the submit button in the form type and then do
> 
> ```
> {{ form_widget(form.submit) }}
> ```
> 
> I think that works. the label can be set in the type as well

I tried getting this to work, but it failed as rendering the form widget will include this button, and I need to place it next to the cancel button.

Reply:
Form_end needs to be after the button. That should fix it